mozanalysis/bootstrap.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
```

Comment:
I recommend to move the source into a `src/` directory to prevent import time issues when the package is located in the root directory and overlaps with installed packages (e.g. during coverage collection). A good primer is located here: https://hynek.me/articles/testing-packaging/